data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1853 @@
+# Copyright (c) 2022 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Lunarium Initial Log"
+	invisible
+	landing
+	to offer
+		has "Lunarium introduced"
+	on offer
+		log "Factions" "Lunarium Redoubt" "The Lunarium Redoubt is a resistance movement within the Coalition that is opposed to the Heliarchs, aiming to free it from their rule."
+		fail
+
+mission "Resistance: Smuggling: Charity 1"
+	name "Charity Delivery"
+	description "Deliver a care package of <cargo> to <destination>, as part of a charity initiative."
+	minor
+	cargo "charity supplies" 22
+	source
+		near "3 Spring Rising" 2 6
+		not government "Heliarch"
+	destination "Fourth Shadow"
+	to offer
+		random < 40
+		has "Coalition: First Contact: done"
+		not "Heliarch License: done"
+		not "Heliarch Investigation 2: active"
+		not "Resistance: Questions: active"
+	on offer
+		conversation
+			branch metchiree
```

Comment:
```suggestion
			branch chiree
```
Trying to use single-word branches/labels where possible (preferable and avoids the discourse in #8753)